styles.css

```diff
@@ -0,0 +1,277 @@
+/*  Sign Up Button */
+body {
+    background: #eceef1;
+}
+div {
+    display: block;
+}
+.sign-up {
+    background: #eceef1;
+    /* background-color: chocolate; */
+    float: none;
+    padding: 10px 30px;
+}
+.s-u-section {
+    float: right;
+    height: 40px;
+    width: 300px;
+}
+.s-u-left {
+    /* background-color: white; */
+    float: left;
+    height: 36px;
+    margin-right: 10px;
+    margin-top: 5px;
+    text-align: right;
+    width: 170px;
+}
+.accnt-msg {
+    font-family: 'Nunito', sans-serif;
+    font-size: 12px;
+    height: 18px;
+    line-height: 1.5px;
+    margin-top: 2px;
+    padding: 1px;
+    text-align: center;
+    vertical-align: auto;
```

Comment:
`REQUIRED` 

`auto` is an invalid value for  `vertical-align` property. Possible values are:

> ```
> baseline, sub, super,  text-top,  text-bottom, middle, top, bottom, 
> <percentage> ( 20%, 30%, ... ), <length>  (10px, 2em, ...) and global values
> ```

[Check here to learn more about `vertical-align`](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align)